### PR TITLE
feat: Groups: Add "Followers" page for Owners/Organizers

### PR DESCRIPTION
**Pull request by MeghalBisht**
Fixes parts of #7601 
To be followed up -  CSV export part

#### Short description of what this resolves:
Screenshot from 2021-07-29 03-38-28
Screenshot from 2021-07-29 03-38-24

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by maze-runnar — DISMISSED** [review]

**Comment by maze-runnar** [comment]
Screenshot from 2021-07-27 22-37-06

**Comment by maze-runnar** [comment]
@MeghalBisht you will create new PR for CSV export, right?

File: app/templates/groups/edit/followers.hbs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<GroupNav @group={{this.model.group}} />
```
Comment (MeghalBisht) [inline app/templates/groups/edit/followers.hbs:1]: @maze-runnar After adding the navigation, if you try to save the event settings or events to the group it throws an error.

Comment (maze-runnar) [reply app/templates/groups/edit/followers.hbs:1]: Oh, I will check and try to fix it. If not possible then will revert.

**Comment by maze-runnar** [comment]
After visiting to followers route, whether the group-nav is added or not , saving event is giving error. And i think followers should not be in edit section.

**Comment by mariobehling** [comment]
I think the slugs we are implementing do now always make sense, but I do not have the perfect solution right now. In any case the followers list will be editable in some ways by organizers, but these are future issues (e.g. organizers should be able to throw out a user). There could also be a public followers list, but again it is a future issue. So, this looks like a good approach to me in general.

**Comment by mariobehling** [comment]
I missed a point in the issue description. We need a "followers icon" behind the "Events" icon on the groups overview page. The followers icon should be different to the team icon. Could you add it, please as well?

Screenshot from 2021-07-28 13-44-11

I see that the page is not working on the test deployment. I guess it is because we are using the production API server for testing. Have you been able to see how the PR frontend deployments on Vercel could be changed to use https://test-api.eventyay.com instead?

**Comment by maze-runnar** [comment]
@MeghalBisht I have changed the path for the followers page and added follower icon. if it's ok, I can push changes?

Screenshot from 2021-07-28 22-25-09
Screenshot from 2021-07-28 22-25-05

**Comment by MeghalBisht** [comment]
> @MeghalBisht I have changed the path for the followers page and added follower icon. if it's ok, I can push changes?

I tried this too. But after you visit follower's page and again try to go back to edit events or settings page, the patch request fails. Did you check that?

**Comment by maze-runnar** [comment]
> > @MeghalBisht I have changed the path for the followers page and added follower icon. if it's ok, I can push changes?
> 
> I tried this too. But after you visit follower's page and again try to go back to edit events or settings page, the patch request fails. Did you check that?
> 
> 

Yeah, error is not solved 😅

**Review by maze-runnar — APPROVED** [review]

**Pull request merged**